job-templates/job.yaml

```diff
@@ -13,21 +13,30 @@ spec:
       containers:
       - name: buildkite-agent
         image: buildkite/agent:latest
-        args: ["start", "--disconnect-after-job", "--disconnect-after-job-timeout", "300"]
+        args:
+          [
+            "start",
+            "--disconnect-after-job",
+            "--disconnect-after-job-timeout",
+            "300",
+          ]
```

Comment:
this is like another commit, but also why?

Reply:
sorry, it was my autoformatter - should have double checked. fixed though!